### ISSUE TITLE
No .exe file in package folder and cornflakes linter complains executable not found in specified folder (json.settings vs code)

### ISSUE THREAD

**Issue by ScoobyQ** [PyCQA/flake8#1596]
### how did you install flake8?

```console
pip install flake8 --user
```

### unmodified output of `flake8 --bug-report`

```json
N/A
```

### describe the problem

I was expecting the linter to work after following the install instructions https://flake8.pycqa.org/en/latest/ and configuration instructions for vs code. However, I get an executable not found error message and indeed I cannot see a .exe in the flake8 package folder. Assuming I have somehow messed something up.

**Comment by asottile** [comment]
there will be a Scripts or bin folder that you need to make sure is on your PATH when you perform a user install (pip should warn you as well)

this isn't a flake8 issue
